### PR TITLE
feat: Add autoindex for Python

### PR DESCRIPTION
**Pull request by tjdevries**
Don't merge yet, this dockerfile doesn't exist yet on docker hub. Just pushing to get feedback from @efritz 

## Test plan

- [ ] Test adding some python deps via `scip-python` (can add more instructions after that's complete)
- [ ] Could write/run some more tests if there are any cases I'm missing

### CODE REVIEW

File: enterprise/cmd/worker/internal/codeintel/indexing/dependency_sync_scheduler.go [hunk 1]
```diff
@@ -90,6 +91,7 @@ func (h *dependencySyncSchedulerHandler) Handle(ctx context.Context, logger log.
 
 	for {
 		packageReference, exists, err := scanner.Next()
+
```
Comment (efritz) [inline enterprise/cmd/worker/internal/codeintel/indexing/dependency_sync_scheduler.go:94]: Why the newline?

File: internal/codeintel/autoindexing/internal/inference/lua/.stylua.toml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+indent_type = "Spaces"
```
Comment (efritz) [inline internal/codeintel/autoindexing/internal/inference/lua/.stylua.toml:1]: 🙏

Comment (efritz) [reply internal/codeintel/autoindexing/internal/inference/lua/.stylua.toml:1]: Is this meant to be capitalized? Looks weird 🤔

**Review by efritz — APPROVED** [review]
I saw this Lua work with my own eyes. 👁️ 👄 👁️

File: internal/codeintel/autoindexing/internal/inference/lua/.stylua.toml (hunk 2 above)
Comment (tjdevries) [reply internal/codeintel/autoindexing/internal/inference/lua/.stylua.toml:1]: it's just how Johnny likes the names haha

**Pull request merged**